### PR TITLE
Link option with title

### PR DESCRIPTION
**Pull request by mgohin**
We can now add a title for links

### CODE REVIEW

**Comment by TerryMooreII** [comment]
Instead of prompting for a title can we get the title from the text that the user has highlighted and if no text is selected then just use the URL?  Eventually I want to have a small modal popup like google docs.

File: src/angular-wysiwyg.js [hunk 1]
```diff
@@ -307,9 +307,10 @@ Requires:
                 }
 
                 scope.createLink = function() {
-                    var input = prompt('Enter the link URL');
-                    if (input && input !== undefined)
-                        scope.format('createlink', input);
+                    var inputUrl = prompt('Enter the link URL');
+                    var inputTitle = prompt('Enter the link title');
+                    if (inputUrl && inputTitle)
+                      document.execCommand('insertHTML', false, '<a href="' + inputUrl + '">' + inputTitle + '</a>');
```
Comment (TerryMooreII) [inline src/angular-wysiwyg.js]: This should be changed to use the scope.format() function.

**Comment by mgohin** [comment]
Ok with your suggestions. I did this very quickly to fit my needs ;)

**Comment by BharatGK** [comment]
how i can set title inside <a> tag like <a href="" title="">original text</a>.
I dont want to replace original text. How i can?
thanks

**Pull request closed without merging**